examples/auth-demo/README.md

```
@@ -0,0 +1,590 @@
+```json
+{
+  "accessToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9...",
+  "expiresIn": 900
+}
+```
+
+#### GET /auth/api/verify
+トークンの検証を行います。
+
+**ヘッダー:**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)